app/config/question-bank.js

```diff
@@ -1702,6 +1702,78 @@ const questionBank = {
           },
           yarKey: "serviceCapacityIncrease",
         },
+        {
+          key: "existing-cover-type",
```

Comment:
This existing-cover-type can be removed (we have another one further down with order number 136)